tests/integration_tests/tests/agentless_tests/test_resume.py

```diff
@@ -335,6 +335,7 @@ def test_force_resume_cancelled_nonresumable(self):
         self.assertEqual(execution.status, 'cancelled')
 
         self._unlock_operation('interface1.op_nonresumable')
+        time.sleep(5)
```

Comment:
I think it happens in more tests :/

Reply:
yep, every test that uses _unlock_operation, so i put the sleep there

(one test already had it)